### PR TITLE
s/handshake protocol/control protocol

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

**Review by mirjak — APPROVED** [review]
If we talk about the control part we should probably make more clear that this is not only the initial handshake but also any control that e.g. needed to refresh keys.

File: draft-kuehlewind-taps-crypto-sep.md [hunk 1]
```diff
@@ -49,12 +49,15 @@ informative:
 
 --- abstract
 
+<!-- TODO(caw): we might consider rewriting this abstract; it no longer applies. -->
```
Comment (mirjak) [inline draft-kuehlewind-taps-crypto-sep.md]: What do you need to change here?

Comment (chris-wood) [reply draft-kuehlewind-taps-crypto-sep.md]: It was stale -- I removed that comment.

**Comment by chris-wood** [comment]
I also added a comment about using the control protocol after the handshake completes to update shared secrets.

**Pull request merged**